instrumentation/aws-lambda/aws-lambda-events-2.2/library/src/test/java/io/opentelemetry/instrumentation/awslambdaevents/v2_2/TracingRequestWrapperStandardEventsTest.java

```diff
@@ -15,198 +15,215 @@
 import com.amazonaws.services.lambda.runtime.events.SNSEvent;
 import com.amazonaws.services.lambda.runtime.events.SQSEvent;
 import com.amazonaws.services.lambda.runtime.events.ScheduledEvent;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import io.opentelemetry.instrumentation.awslambdacore.v1_0.internal.WrappedLambda;
+import io.opentelemetry.instrumentation.awslambdaevents.v2_2.internal.SerializationUtil;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
+import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
 class TracingRequestWrapperStandardEventsTest {
-  private static final String SUCCESS = "success";
-  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-  private static final Map<Class<?>, String> EVENTS_JSON = buildEventExamples();
+  private static final Map<Class<?>, EventInfo> EVENTS_JSON = buildEventExamples();
 
   private final OpenTelemetrySdk sdk = OpenTelemetrySdk.builder().build();
   private final Context context = mock(Context.class);
   private TracingRequestWrapper wrapper;
 
-  private static Map<Class<?>, String> buildEventExamples() {
-    Map<Class<?>, String> events = new HashMap<>();
+  static final class EventInfo {
+    final Class<?> eventType;
+    final String eventBody;
+
+    EventInfo(Class<?> eventType, String eventBody) {
+      this.eventType = eventType;
+      this.eventBody = eventBody;
+    }
+  }
+
+  private static Map<Class<?>, EventInfo> buildEventExamples() {
+    Map<Class<?>, EventInfo> events = new HashMap<>();
```

Comment:
Probably want to use `ClassValue` here as well for consistency and the reasons @tylerbenson mentioned previously.

Reply:
I'm less concerned about it in a test class, but 🤷.

Reply:
I couldn't get it how to use `ClassValue` here. There is no lazy retrieved value here and not sure `ClassValue` fits better here than `Map`

Reply:
Ah yeah ok test class, it's probably fine.